setup.py

```diff
@@ -20,35 +19,26 @@ def read(f):
 
 
 def read_version():
-    regexp = re.compile(r"^__version__\W*=\W*'([\d.abrc]+)'")
-    init_py = os.path.join(os.path.dirname(__file__),
-                           'aiomonitor', '__init__.py')
-    with open(init_py) as f:
-        for line in f:
-            match = regexp.match(line)
-            if match is not None:
-                return match.group(1)
-        else:
-            msg = 'Cannot find version in aiomonitor/__init__.py'
-            raise RuntimeError(msg)
+    return version('aiomonitor')
```

Comment:
This wouldn't work because this function reads the version of an installed distribution and at this point it's not yet installed, not even built.

Reply:
Also, now that this function isn't called at all, it's not needed either. Just drop it.